Gemfile.lock

```diff
@@ -119,4 +121,4 @@ DEPENDENCIES
   rouge (~> 2.0.5)
 
 BUNDLED WITH
-   1.14.3
+   1.15.0
```

Comment:
I don't think we should be changing internal files required by the API-docs framework